src/Command/ProxySyncReleasesCommand.php

```diff
@@ -59,6 +77,7 @@ private function syncPackages(\SimpleXMLElement $feed): void
         }
 
         foreach ($feed->channel->item as $item) {
+            $this->lock->refresh();
```

Comment:
I think `refresh` can happen in `if (isset($syncedPackages[$name])) {` condition

Reply:
done